cmd/cerbosctl/enable/internal/policy/policy.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2021-2023 Zenauth Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+package policy
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/cerbos/cerbos/client"
+	"github.com/cerbos/cerbos/cmd/cerbosctl/internal"
+)
+
+func Enable(c client.AdminClient, ids ...string) (uint32, error) {
+	var enabledPolicies uint32
+	for idx := range ids {
+		if idx%internal.MaxIDPerReq == 0 {
+			idxEnd := internal.MinInt(idx+internal.MaxIDPerReq, len(ids))
+			var err error
+			enabled, err := c.EnablePolicy(context.Background(), ids[idx:idxEnd]...)
+			if err != nil {
+				return 0, fmt.Errorf("error while enabling policy: %w", err)
+			}
+			enabledPolicies += enabled
+		}
+	}
```

Comment:
I see this batching pattern being repeated a lot. Now that Go generics support has landed, I wonder if it could be made into a utility function that can be reused.